Client/Client.Qt/Widgets/MessageWidget.cpp

```diff
@@ -67,7 +75,8 @@ void MessageWidget::paintEvent(QPaintEvent* e)
     }
 
     auto usernameRect = QRect(margin * 2, margin * 2 + 1, getStyle().fontname->width(getUserName()), getStyle().fontname->height);
-    p.drawText(usernameRect, getUserName());
+
+    // p.drawText(usernameRect, getUserName());
```

Comment:
I would get rid of the commented lines that are not used in the code

Reply:
While this is a draft, it's okey